src/main/java/hw_5/Task2/MyInt.java

```diff
@@ -0,0 +1,14 @@
+package hw_5.Task2;
+
+public class MyInt {
+    public int x;
+
+
+    public MyInt(int x) {
+        this.x = x;
+    }
+    @Override
```

Comment:
there is should be extra empty line between 9 and 10 rows

Reply:
fixed